.flowconfig

```diff
@@ -45,15 +45,14 @@ module.file_ext=.js
 module.file_ext=.jsx
 module.file_ext=.json
 
-module.name_mapper='^image![a-zA-Z0-9$_-]+$' -> 'GlobalImageStub'
```

Comment:
React deprecated `require('image!./img.png')` support a long time ago, and they just completely removed it.